test/integration/scenarios/network/peer_disconnect.js

```diff
@@ -144,22 +133,11 @@ module.exports = params => {
 
 				describe('after all the node restarts', () => {
 					before(done => {
-						utils.ws.establishWSConnectionsToNodes(
-							params.configurations,
-							(err, socketsResult) => {
-								if (err) {
-									return done(err);
-								}
-								params.sockets = socketsResult;
-								network.enableForgingOnDelegates(params.configurations, () => {
-									done();
-								});
-							}
-						);
+						network.enableForgingForDelegates(params.configurations, done);
 					});
 
-					// The expected connection becomes 180(new connection) + 18 (previously held connections)
-					it(`should be ${expectedOutgoingConnections +
+					// The expected connection becomes 'networkParameters.EXPECTED_OUTOGING_CONNECTIONS' + 18 previously held connections
```

Comment:
Why + 18? We restarted all nodes in the network?

Reply:
Same as above

Reply:
It can be added as a separate issue as the next following improvements -- explain hardcoded variables.